Sources/GDSCommon/Patterns/GDSInformation/GDSInformationViewModel.swift

```diff
@@ -28,12 +28,24 @@ public protocol GDSInformationViewModel {
     var imageHeightConstraint: CGFloat? { get }
     var title: GDSLocalisedString { get }
     var body: GDSLocalisedString? { get }
-    var footnote: GDSLocalisedString? { get }
+}
+
+@MainActor
+public protocol GDSInformationViewModelFootnote {
```

Comment:
how many places is this used? Can we add typealiases / deprecate things to prevent this from being a breaking change?

https://govukverify.atlassian.net/wiki/spaces/DCMAW/pages/4391305315/Mitigating+breaking+change